awsutil/awsutil.go

```diff
@@ -29,6 +31,15 @@ func (e ECSTaskMeta) TaskID() string {
 	return split[len(split)-1]
 }
 
+func (e ECSTaskMeta) Region() string {
+	// https://github.com/aws/containers-roadmap/issues/337
+	split := strings.Split(e.TaskARN, ":")
+	if len(split) < 4 {
+		return ""
```

Comment:
should we return error? If the AWS SDK client will not function without a region then would it make sense to fail fast?